test/unit_tests/modules/util/type-checkers-spec.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+var chai = require( 'chai' );
+var expect = chai.expect;
+var typeCheckers = require(
+  '../../../../src/static/js/modules/util/type-checkers.js'
```

Comment:
At some point it might make sense to make a test paths module that has the JS source base path, so that this could look something like:

```
var config = require( 'config' );
var typeCheckers = require( config.jsSrcPath + 'modules/util/type-checkers.js' );
```

Perhaps could be covered in https://github.com/cfpb/cfgov-refresh/issues/861